.github/pull_request_template.md

```diff
@@ -16,22 +16,9 @@ This should be a short TL;DR that includes the purpose of the PR.
 
 <!-- Issues, RFC, etc. -->
 
-
-## :building_construction: How to Build and Test the Change
-
-<!-- List steps to test your change on a local environment. -->
-
-
-## :+1: Definition of Done
-
-- [ ] New functionality works 
-- [ ] Accessibility addressed
-- [ ] Tests added
-- [ ] Docs updated
-
-### :speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.
+## :speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.
```

Comment:
[suggestion] I would streamline this even more:
- just have this as a normal paragraph (separated from the content above by an horizontal divider
- remove the examples below

Reply:
dunzo! and updated in the PR description